sdk/typescript/src/utils/format.ts

```diff
@@ -13,5 +13,5 @@ export function formatAddress(address: string) {
 
 export function formatDigest(digest: string) {
   // Use 10 first characters
-  return `${digest.slice(0, 10)}${ELLIPSIS}`;
+  return `${digest.slice(0, 10)}${ELLIPSIS}${digest.slice(-4)}`;
```

Comment:
This isn't correct, this is the format that we're expecting (design might be out of date).

Reply:
Reverted back to og values